doctr/models/detection/core.py

```diff
@@ -35,7 +35,7 @@ def __init__(
         self,
         output_size: Tuple[int, int],
         batch_size: int = 1,
-        mean: Tuple[float, float, float] = (.5, .5, .5),
+        mean: Tuple[float, float, float] = (0.5, 0.5, 0.5),
```

Comment:
No need for the zeros before the '.' (so that the PR doesn't display any diff on this file)